elements/pl-matching/pl-matching.py

```diff
@@ -0,0 +1,438 @@
+import prairielearn as pl
+import lxml.html
+import random
+import math
+import chevron
+
+
+WEIGHT_DEFAULT = 1
+FIXED_STATEMENTS_ORDER_DEFAULT = False
+FIXED_OPTIONS_ORDER_DEFAULT = False
+INLINE_DEFAULT = False
+PARTIAL_CREDIT_DEFAULT = True
+HIDE_ANSWER_PANEL_DEFAULT = False
+HIDE_HELP_TEXT_DEFAULT = False
+DETAILED_HELP_TEXT_DEFAULT = False
+HIDE_SCORE_BADGE_DEFAULT = False
+BLANK_DEFAULT = True
+BLANK_ANSWER = ' '
```

Comment:
I believe this is now -1, right?

Reply:
This is the literal string shown in the `select` list.